beta/src/pages/learn/state-a-components-memory.md

```diff
@@ -1,25 +1,25 @@
 ---
-title: "State: A Component's Memory"
+title: "State: एक कौम्पोनॅन्ट की मेमोरी"
 ---
 
 <Intro>
 
-Components often need to change what's on the screen as a result of an interaction. Typing into the form should update the input field, clicking "next" on an image carousel should change which image is displayed, clicking "buy" should put a product in the shopping cart. Components need to "remember" things: the current input value, the current image, the shopping cart. In React, this kind of component-specific memory is called *state*.
+कोई भी इंटरेक्शन होने की वजह स कौम्पोनॅन्ट को अक्सर स्क्रीन पर क्या दिख रहा है उसको बदलना होता है। फॉर्म में टाइप करने पर इनपुट फील्ड अपडेट होने चाहिए, इमेज करौसेल के "next" को क्लिक करने पर करौसेल की इमेज बदलनी चाहिए, "buy" पर क्लिक करने पर एक प्रोडक्ट शॉपिंग कार्ट में ऐड होना चाहिए। कौम्पोनॅन्ट को चीजों को "याद रखना" चाहिए: इनपुट की अभी की वैल्यू, अभी की इमेज, शॉपिंग कार्ट। React में, इस प्रकार की कौम्पोनॅन्ट-स्पेसिफिक मेमोरी को *State* कहा जाता है।
 
 </Intro>
 
 <YouWillLearn>
 
-* How to add a state variable with the [`useState`](/apis/usestate) Hook
-* What pair of values the `useState` Hook returns
-* How to add more than one state variable
-* Why state is called local
+* state वेरिएबल को [`useState`](/apis/usestate) Hook के साथ कैसे ऐड करें
+* `useState` Hook कोनसे वैल्यूज की पेअर को रिटर्न करता है 
+* एक से अधिक स्टेट वेरिएबल कैसे ऐड करें
+* state को लोकल क्यों कहा जाता है
 
 </YouWillLearn>
 
-## When a regular variable isn’t enough {/*when-a-regular-variable-isnt-enough*/}
+## जब एक सामान्य वेरिएबल पर्याप्त नहीं है {/*जब एक सामान्य वेरिएबल पर्याप्त नहीं है*/}
```

Comment:
`{/*जब एक सामान्य वेरिएबल पर्याप्त नहीं है*/}` => `{/*when-a-regular-variable-isnt-enough*/}`

let's not translate these. These are used for reflinks

Reply:
Oh okay, sure!